### DISCUSSION TITLE
Hotkey customization?

### DISCUSSION THREAD

**Discussion by bangoobangoo** [d2phap/ImageGlass#2014] [Q&A] [answered]
Sorry if this is obvious-- am I missing something, or can I not set my own hotkeys for ImageGlass? "Customizing keyboard and mouse actions to your preferences" is listed under Other Features on the website, but I can't find anything in Settings besides mouse wheel options.

I've really liked using IG as a default image viewer for a couple years now, but I just installed on a new machine and seem to remember on an older version that Esc would allow me to exit fullscreen instead of close the entire window? Appreciate any help.

**Comment by d2phap** [comment, marked as answer]
The hotkey customization is supported (experiment) via config file only, you can refer to https://github.com/d2phap/ImageGlass/discussions/1702#discussioncomment-7660806 for the details

**Reply by bangoobangoo** [reply]
Appreciate the response. I'm apparently unable to bind the Esc key however, because it closes the "Add hotkey" window on being pressed. :D I'll just put up with it as is until custom keybinds are added to the UI. Thanks again!

**Reply by Valour549** [reply]
@bangoobangoo The name of the Esc key is just **Escape**

So you can do something like the following so Esc toggles in and out of fullscreen instead of F11
`"MenuHotkeys": { "MnuFullScreen": ["Escape"] }`
